taskpools/chase_lev_deques.nim

```diff
@@ -113,13 +111,21 @@ proc grow[T](deque: var ChaseLevDeque[T], buf: var ptr Buf[T], top, bottom: int)
   for i in top ..< bottom:
     tmp[][i] = buf[][i]
 
-  buf.prev = deque.garbage
+  buf.prevRetired = deque.garbage
   deque.garbage = buf
   # publish globally
   deque.buf.store(tmp, moRelaxed)
   # publish locally
   swap(buf, tmp)
 
+proc garbageCollect(deque: var ChaseLevDeque) {.inline.} =
```

Comment:
```suggestion
proc garbageCollect(deque: var ChaseLevDeque) =
```

there's no difference between inline and not on private functions

Reply:
Actually there is, you can access private functions and variables from another module, using templates for example.

I'm not too sure how strong a hint `visibility((hidden))`  is to the compiler but it's definitely less than C `static` (only accessible from this module) and `static inline`.